core/src/main/scala/io/finch/internal/ToResponse.scala

```diff
@@ -35,4 +37,18 @@ object ToResponse {
 
       rep
     }
+
+  /**
+   * Provides implicit ToResponse for `AsyncStream[Buf]` since vanilla [[Endpoint]] is synchronous by it's nature.
```

Comment:
I'm not sure I understand what does "since vanilla [[Endpoint]] is synchronous by it's nature" mean. `Endpoint`s are as asynchronous as they could be since a value they return wrapped within a `Future`. So, let's just leave that out for now.